src/ResourceManager/ServiceFabric/Commands.ServiceFabric/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.WebKey" version="2.0.6" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.ServiceFabric" version="1.0.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ServiceFabric" version="1.0.1-preview" targetFramework="net45" />
```

Comment:
@maddieclayton I think the build failure is coming from having `net45` instead of `net452` for all files updating the ServiceFabric package